modular_skyrat/master_files/code/datums/quirks/positive_quirks/life_savings.dm

```diff
@@ -1,20 +1,37 @@
 /datum/quirk/life_savings
-    name = "Life Savings"
```

Comment:
Why are these marked as changed, is this a line ending moment? Or is it the indentation. It looks more indented in your version

Reply:
Yes, it was previously using spaces, now it's tabs

Reply:
Exactly and it made it not compile lol